<?xml version="1.0"?>
<launch>
  <!-- Command-line arguments -->
  <arg name="ns" default="o3d3xx"/>
  <arg name="nn" default="camera"/>
  <arg name="infile" default="-"/>

  <!-- TODO: Consider removing the group and let the includer determine the
      namespace using a group. -->
  <group ns="$(arg ns)/$(arg nn)">
    <node pkg="o3d3xx"
          type="o3d3xx_config_node"
          name="config_node"
          output="screen">

      <param name="infile" value="$(arg infile)"/>
      <remap from="/Config" to="Config"/>

    </node>
  </group>

</launch>
